.storybook/utils.js

```diff
@@ -194,7 +194,7 @@ export function autoPropKnobs(properties, bridge) {
 
         // If this is not a required field, add a null option
         if (!required) {
-          opts.null = "";
+          opts[""] = "default";
```

Comment:
@kylebuch8 I'm fine if the printed value is default but the stored value should stay `null` for the rest of the logic in the dynamic build to work.

Reply:
i.e., `opts.null = "default"`